applications/vanilla/modules/class.tagmodule.php

```diff
@@ -103,6 +103,15 @@ public function getData() {
 
         $this->autoContext();
 
+        $tagData = null;
+        $this->EventArguments['ParentID'] = $this->ParentID;
```

Comment:
ParentType is available and used by this method, so it's probably worth passing along too.